browser/ipfs/ipfs_tab_helper.cc

```diff
@@ -262,30 +270,38 @@ void IPFSTabHelper::MaybeCheckDNSLinkRecord(
     const net::HttpResponseHeaders* headers) {
   UpdateDnsLinkButtonState();
   auto current_url = GetCurrentPageURL();
+  auto dnslink_target = current_url;
```

Comment:
Can you explain why dnslink_target should be initialized to LastCommitedURL?

Reply:
All IPFS-related checks are initiated after navigation commit, so we use committed url for them.